### ISSUE TITLE
Update meta description

### ISSUE THREAD

**Issue by EmmaDawsonDev** [AccessibleForAll/AccessibleWebDev#29]
In pages/index.tsx the current meta description on line 12 is ```<meta name="description" content="Generated by create next app" />```

This should be updated to ```<meta name="description" content="Learn the basics about web accessibility in a clear and easy to understand way" />```

**Comment by cyrilckurian** [comment]
hello ,i would like to work on this issue can you assign this to me @EmmaDawsonDev

**Comment by EmmaDawsonDev** [comment]
Sure, go ahead :) If you're taking part in hacktoberfest then just wait until October 1st before making the pull request.

**Comment by cyrilckurian** [comment]
Yes, I'm taking part in hackoberfest
